nixos/modules/system/activation/activation-script.nix

```diff
@@ -128,14 +128,6 @@ in
       ''
         # Various log/runtime directories.
 
-        mkdir -m 0755 -p /run/nix/current-load # for distributed builds
```

Comment:
This isn't done anywhere else is it?

Reply:
In the commit message it says that this was used in an very old version of nix.

Reply:
Ok thanks. Missed the commit messages but they are very informative.